extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -1750,6 +1763,29 @@ export class CodeQLCliServer implements Disposable {
     this._versionChangedListeners.push(listener);
   }
 
+  private async notifyPackChanged(packDir: string) {
+    const packFilePath = join(packDir, "codeql-pack.yml");
```

Comment:
I believe pack files can also be located at `qlpack.yml` for legacy pack files. You should be able to use the `getQlPackFilePath` function to get the correct pack file.

Reply:
Thanks for pointing that out (now I know!). But addressing your other comment "Do you still need this if you are also restarting the language client?" meant this logic was removed altogether :)